src/components/shared/signOut/SignOut.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+
+import { NavLink } from '@mantine/core';
+import { IoIosLogOut } from 'react-icons/io';
+
+import { useAppDispatch } from 'store/store';
+import { userThunk } from 'store/user/userData.api';
+
+const SingOutButton: FC = () => {
+  const dispatch = useAppDispatch();
+  const handleClick = async (): Promise<void> => {
+    dispatch(userThunk.signOut());
+  };
```

Comment:
what is async function for?)

Reply:
The answer is here https://github.com/Egorkostuck/project/pull/2#discussion_r1477135947